HelpSource/Guides/MultiClient_Setups.schelp

```diff
@@ -0,0 +1,176 @@
+TITLE:: MultiClient_Setups
+SUMMARY:: How to handle clientIDs and id allocation in multi-client/server setups
+CATEGORIES:: Guides
+related:: Classes/ServerOptions, Classes/Server
+KEYWORD:: server, multi-client
+
+section:: Multi client-server setups - discussion and tests
+
+OSC communication between SC and its sound server offers many options for network music: Multiple computers can run both supercollider and associated sound servers.
+For clarity, the "server program" refers a running scsynth or supernova process. The "server object" AKA "client" is the server's representation in sclang, such as code::Server.local, s, Server(\elsewhere, NetAddr("123.234.56.78"))::.
+
+subsection:: What are clientIDs for, and how do servers get them?
+
+When more than one user plays on a given server,
+some resources need to be shared between users/clients:
+list::
+## permanent and temporary nodeIDs (handled by s.nodeAllocator),
+## private control and audio bus channels (handled by s.audioBusAllocator, s.controlBusAllocator)
+## buffer numbers (handled by s.bufferAllocator).
+::
+
+In the simple default case, there is only a single user/client, who always gets clientID 0, and all the available resources (i.e. the full ranges of allocator numbers).
+
+When multiple clients log in, this is what happens:
+list::
+## On startup, scsynth (the server program) gets a fixed limit of maxLogins.
+## When a new local or remote server object/client registers on this scsynth process,
+scsynth sends back the next free clientID.
+## Exception: If the server object was given a clientID on creation, it keeps that clientID, to allow fixed clientID distribution. (This variant may become deprecated in the future.)
+::
+
+subsection:: Code examples and tests
+
+A. Test the clientID method by hand
+
+Note:: Setting clientID_ by hand is not recommended, as it can be confusing.
+Here it is only done for testing the method!::
+
+code::
+
+s.options.maxLogins = 12;
+// these work and post info
+s.clientID_(1);
+s.clientID_(2);
+s.clientID_(11);
+// guards against bad input - warns and keeps clientID
+s.clientID_(nil);
+s.clientID_("abc");
+s.clientID_(16);
+s.clientID_(-234);
+// reset to default:
+s.clientID_(0);
```

Comment:
Since it is not recommended usage, all this should not be in the guide but rather in a `UnitTest`,